protobuf/flow/entities/block_header.proto

```diff
@@ -14,7 +14,8 @@ message BlockHeader {
   google.protobuf.Timestamp timestamp = 4;
   bytes payload_hash = 5;
   uint64 view = 6;
-  repeated bytes parent_voter_ids = 7; // deprecated!! value will be empty. replaced by parent_vote_indices
+  repeated bytes parent_voter_ids =
```

Comment:
this is some really weird formatting. Is this ok?

Reply:
this is auto from `make format`, I'll undo this for pragmatism